tests/messages/xml/test_pos.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import annotations
+
+import pytest
+
+from deebot_client.events import Position, PositionsEvent
+from deebot_client.message import HandlingState
+from deebot_client.messages.xml import Pos
+from deebot_client.rs.map import PositionType
+from tests.messages import assert_message, assert_message_failure
+
+
+@pytest.mark.parametrize("position", [(-9, 15, 89)])
+def test_Pos(position: tuple[int, int, int]) -> None:
+    x, y, a = position
+    xml_message = f'<ctl td="Pos" t="p" p="{x},{y}" a="{a}" valid="1" />'
```

Comment:
Some questions about the message:
- is `t` the type and will it be different for the charger position?
- There is a `valid` attribute... I would assume that the position is only valid if it is equal to 1. Can you please verify it

Reply:
Fair point
- I have no way of checking if valid = 0 ever happens, but I guess one more check doesn't hurt.
- The app does show the charger position, I just didn't figure out what param it uses. I'll check that tomorrow

Reply:
There's a GetChargerPos command, but I didn't find any "Pos" events for it.
Suggestion:
- Imlpement GetChargerPos 
- Only handle valid="1" t="p" messages as bot position, we'll leave everything else as HandlingResult.analyse() so that they can be eventually debugged further